src/tailwind/defaultConfig.ts

```diff
@@ -284,6 +284,7 @@ export default {
     letterSpacing: {
       normal: "0",
       wide: "1px",
+      wider: "2px",
```

Comment:
next up: widest!

Reply:
haha, that could be and after that we end up with `extra-turbo-mega-widest` :D

In all seriousness though - maybe we should revert to `2xs`-`xs`-`s`-`md`-lg`-`xl`-2xl` scale, which seems to be tailwind default. Fred had been asking if there's a way to change the spacing scale to `8`-`16`-`24`-`32`-`40`-`48`and that's going to be a huge pain if we keep using pixel values in the names...